command/v7/continue_deployment_command_test.go

```diff
@@ -202,6 +207,36 @@ var _ = Describe("Continue deployment command", func() {
 						Expect(executeErr).ToNot(HaveOccurred())
 					})
 
+					When("the --no-wait flag is not provided", func() {
+						BeforeEach(func() {
+							noWait = false
+						})
+
+						It("polls and waits", func() {
+							Expect(fakeActor.PollStartForDeploymentCallCount()).To(Equal(1))
+
+							var invokedApplication, invokedGuid, invokedNoWait, _ = fakeActor.PollStartForDeploymentArgsForCall(0)
+							Expect(invokedApplication).To(Equal(returnedApplication))
+							Expect(invokedGuid).To(Equal(deploymentGUID))
+							Expect(invokedNoWait).To(Equal(false))
+						})
+					})
+
+					When("the --no-wait flag is provided", func() {
+						BeforeEach(func() {
+							noWait = true
+						})
+
+						It("polls without waiting", func() {
+							Expect(fakeActor.PollStartForDeploymentCallCount()).To(Equal(1))
+
+							var invokedApplication, invokedGuid, invokedNoWait, _ = fakeActor.PollStartForDeploymentArgsForCall(0)
```

Comment:
*nit/formatting:*
```suggestion
							invokedApplication, invokedGuid, invokedNoWait, _ := fakeActor.PollStartForDeploymentArgsForCall(0)
```

Reply:
Addressed in 0e2606a20ed0256a052eda1275f1d3e50e56d80d.